apps/guide/src/content/02-installations-and-preparations/02-setting-up-a-linter.mdx

```diff
@@ -0,0 +1,6 @@
+---
+title: Setting up a linter
+category: Installations and preparations
+---
+
+TODO: Rewrite. Placeholder page for ordering.
```

Comment:
#9412 already has a rewritten linter page [here](https://github.com/discordjs/discord.js/pull/9412/files#diff-5574e1e0550d17fb6401ec869d7e44adcd9c8ce79a7c0caa07d989d49301c739R6). Can we use that?

Reply:
That looks much more like you've simply copied the existing one than rewritten it, so no.